package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node .",
-    "test": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test mocha test/*",
```

Comment:
*I think* this is due to the fact that you have a `lib` folder in your `test` folder. You could remove the `lib` folder or add `mocha --recursive`.